var/spack/repos/builtin/packages/spla/package.py

```diff
@@ -58,4 +59,23 @@ def cmake_args(self):
             args += ["-DSPLA_STATIC=ON"]
         else:
             args += ["-DSPLA_STATIC=OFF"]
+
+        if self.spec['blas'].name == 'openblas':
+            args += ["-DSPLA_HOST_BLAS=OPENBLAS"]
+
+        if self.spec['blas'].name in ['amdblis', 'blis']:
```

Comment:
I would use `elif` for these

Reply:
Thanks for the suggestion, I've changed it to `elif`